packages/language/src/v1beta1/language_service_api.js

```diff
@@ -0,0 +1,249 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * EDITING INSTRUCTIONS
+ * This file was generated from the file
+ * https://github.com/googleapis/googleapis/blob/master/library.proto,
+ * and updates to that file get reflected here through a refresh process.
+ * For the short term, the refresh process will only be runnable by Google
+ * engineers.
+ *
+ * The only allowed edits are to method and file documentation. A 3-way
+ * merge preserves those additions if the generated source changes.
+ */
+/* TODO: introduce line-wrapping so that it never exceeds the limit. */
+/* jscs: disable maximumLineLength */
+'use strict';
+
+var arguejs = require('arguejs');
+var configData = require('./language_service_client_config');
+var gax = require('google-gax');
+
+/** The default address of the service. */
+var SERVICE_ADDRESS = 'language.googleapis.com';
+
+/** The default port of the service. */
+var DEFAULT_SERVICE_PORT = 443;
+
+var CODE_GEN_NAME_VERSION = 'gapic/0.1.0';
+
+var DEFAULT_TIMEOUT = 30;
+
+/**
+ * The scopes needed to make gRPC calls to all of the methods defined in
+ * this service.
+ */
+var ALL_SCOPES = [
+  'https://www.googleapis.com/auth/cloud-platform'
+];
+
+module.exports = function build(options) {
+  options = options || {};
+  if (!('scopes' in options)) {
+    options.scopes = ALL_SCOPES;
+  }
+  var gaxGrpc = gax.grpc(options);
+
+  var grpcClient = gaxGrpc.load([{
+    root: require('google-proto-files')('..'),
+    file: 'google/cloud/language/v1beta1/language_service.proto'
+  }]);
+  var built = grpcClient.google.cloud.language.v1beta1;
+  /**
+   * Provides text analysis operations such as sentiment analysis and entity
+   * recognition.
+   *
+   * @class
+   * @param {?Object} opts - The optional parameters.
+   * @param {String} opts.servicePath
+   *   The domain name of the API remote host.
+   * @param {number} opts.port
+   *   The port on which to connect to the remote host.
+   * @param {grpc.ClientCredentials} opts.sslCreds
+   *   A ClientCredentials for use with an SSL-enabled channel.
+   * @param {Object} opts.clientConfig
+   *   The customized config to build the call settings. See
+   *   {@link gax.constructSettings} for the format.
+   * @param {number} opts.timeout
+   *   The default timeout, in seconds, for calls made through this client.
+   * @param {number} opts.appName
+   *   The codename of the calling service.
+   * @param {String} opts.appVersion
+   *   The version of the calling service.
+   */
+  function LanguageServiceApi(opts) {
```

Comment:
Can you move `LanguageServiceApi` from outside of the `build()` function? Having it this way causes it to be re-created each time `build()` is called.